Makefile.build

```diff
@@ -298,6 +298,22 @@ push-http-git-server:
 	@gcloud $(GCLOUD_QUIET) auth configure-docker $(firstword $(subst /, ,$(E2E_TEST_IMAGE_HTTP_GIT_SERVER)))
 	@docker push $(E2E_TEST_IMAGE_HTTP_GIT_SERVER)
 
+# The tinyproxy Docker image is used in e2e/testdata/proxy/tinyproxy.yaml.
+TINYPROXY_SOURCE := kalaksi/tinyproxy:1.6
```

Comment:
It looks like image is maintained by an individual user - is that right? Why is this chosen as the source image?

Reply:
Yes. It seems to be maintained by an individual user: https://github.com/kalaksi/docker-tinyproxy. 

I picked this one from Docker Hub since it has been updated recently and has been downloaded about 100K+ times: https://hub.docker.com/search?q=tinyproxy&sort=updated_at&order=desc.